docs/spark/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Spark
+description: Spark top level. This section priovids indexing for finding how to integrate lakeFS with Spark. As well as using dedicated table formats such as hudi, iceberg and deltalake.
+nav_order: 8
+has_children: false
+---
+
+
+# lakeFS with Spark:
+{: .no_toc }
+
+## Looking to run lakeFS with Apache Spark 💫 ?  
+
+Visit [Spark-lakeFS integration page](../integrations/spark.md).
+
+Want to integrate lakeFS with Spark and DeltaLake? Checkout [Configuring DeltaLake with lakeFS](../integrations/delta.md).
```

Comment:
This is pretty specific... I think that the mapping should be the other way around, i.e. figure out which integration pages are most likely related to Spark and link (VISIBLY) to the lakeFS Hadoop Filesystem integration page.

Reply:
Thanks, Jonathan, do you mean to link to this - [#two-tiered-spark-support](https://docs.lakefs.io/integrations/spark.html#two-tiered-spark-support) instead of the whole spark page?

Reply:
updated! Please check now.